DEFAULT_CONFIG.json5

```diff
@@ -413,25 +413,15 @@
               /// The maximum time in microseconds to wait for an available batch before dropping a droppable message if still no batch is available.
               wait_before_drop: 1000,
               /// The maximum deadline limit for multi-fragment messages.
-              /// When sending multi-fragment message, for each consecutive fragment the deadline from
-              /// the start point of message transmission is calculated as 
-              /// wait_before_drop * 2 ^ min(N, ttl) where N is a fragment number.
-              /// Thus, this parameter allows setting a limitation for maximum time for multi-fragment
-              /// message transmission as wait_before_drop * 2 ^ ttl
-              ttl: 8,
+              max_wait_before_drop_fragments: 255000,
             },
             /// Behavior pushing CongestionControl::Block messages to the queue.
             block: {
               /// The maximum time in microseconds to wait for an available batch before closing the transport session when sending a blocking message
               /// if still no batch is available.
               wait_before_close: 5000000,
               /// The maximum deadline limit for multi-fragment messages.
-              /// When sending multi-fragment message, for each consecutive fragment the deadline from
-              /// the start point of message transmission is calculated as 
-              /// wait_before_close * 2 ^ min(N, ttl) where N is a fragment number.
-              /// Thus, this parameter allows setting a limitation for maximum time for multi-fragment
-              /// message transmission as wait_before_close * 2 ^ ttl
-              ttl: 2,
+              max_wait_before_close_fragments: 15000000,
```

Comment:
I don't think it makes sense to have a special treatment for the `wait_before_close`. And 15 seconds seems a bit too much anyway.

Reply:
Well, this is CC::Block functionality, which has basic wait period of 5 seconds. I guess even 60 seconds will be ok here.

Reply:
Do we really want to treat this case differently? To me the `wait_before_close` is more a *I didn't manage to send even 1 message*. So, as long as every fragment is sent within *5s* we are good and there is no need to set yet another option.

Reply:
This is the same question as the one that was the root cause to have this PR. What if I succeeded sending 100 fragments spending 1 second on each one and then spent 6 seconds on sending 101th? Should I treat this as the condition to fail and throw to the garbage 100 seconds of occupied link?
I think the same rule as for Drop is applied here: the more progress we have, the more expensive the cancellation will be.

Reply:
Well, here is not about just dropping the message but closing the session. It's a safeguard about unresponsive readers and I don't think it depends on wether we are fragmenting or not.

Reply:
Moreover, in which way `wait_before_close` is different from the `session lease`?

Reply:
Right! I removed non-linear part for Block